client/signup/steps/get-dot-blog-plans/index.jsx

```diff
@@ -9,7 +9,10 @@ const GetDotBlogPlansStep = ( { queryObject, ...props } ) => (
 	<PlansStep
 		additionalStepData={ {
 			isPurchasingItem: true,
-			domainItem: cartItems.domainMapping( { domain: queryObject.domain } ),
+			domainItem: cartItems.domainMapping( {
+				domain: queryObject.domain,
+				source: 'getdotblog'
```

Comment:
Good point. What about `get-dot-blog-signup`, as the flow name is `get-dot-blog`?